appendix-basic65.tex

```diff
@@ -3447,7 +3456,7 @@ \subsection{ERASE}
    between erasing and recovery, which may have altered the
    contents of the file.
 
-\item [Remarks:] {\bf ERASE filename} works similarly to {\bf SCRATCH filename}.
+\item [Remarks:] {\bf ERASE} works similarly to {\bf SCRATCH}.
```

Comment:
ERASE doesn't work similar, but identical to SCRATCH.
It's an alias and calls the same routine.
DELETE filename
ERASE filename
SCRATCH filename
are different names for the same action.

Reply:
updated them all to say `is a synonym for`